### PR TITLE
chore(web): Add storytelling static content of left panel 

### PR DESCRIPTION
**Pull request by isoppp**
# Overview

I replaced the left side panel in the story tab with actual contents(static).

## What I haven't done

- connect with state/API which means static now
- popup contents
- styles like `selected` or `isActive`

## How I tested

- storybook for some components
  - https://deploy-preview-565--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-sidepanel--default
  - https://deploy-preview-565--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-tabs-story-sidepanel-action--default
  - https://deploy-preview-565--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-tabs-story-sidepanel-item--default
  - https://deploy-preview-565--reearth-web.netlify.app/storybook/?path=/story/beta-features-editor-tabs-story-sidepanel-pageitemwrapper--default
- editor ( need to open beta project )

image

### CODE REVIEW

File: web/src/beta/components/Icon/Icons/square.svg
Comment (isoppp) [inline web/src/beta/components/Icon/Icons/square.svg:1]: updated from beta design system

File: web/src/beta/components/Icon/Icons/two-rectangle.svg
Comment (isoppp) [inline web/src/beta/components/Icon/Icons/two-rectangle.svg:1]: This is old swiper which is added in this PR. Updated and followed design name definition.

File: web/src/beta/features/Editor/Tabs/Storytelling/StorySidePanel/index.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  onSelectPage: (id: string) => void;
+  onPageAdd: () => void;
+};
+
+const StorySidePanel: FC<Props> = ({ onStoryAdd, onSelectStory, onPageAdd, onSelectPage }) => {
+  return (
+    <SidePanel.Wrapper location="left">
+      <SidePanel.Section maxHeight="33%">
+        <SidePanel.Card>
+          <SidePanel.Title>Story</SidePanel.Title>
```
Comment (KaWaite) [inline web/src/beta/features/Editor/Tabs/Storytelling/StorySidePanel/index.tsx:25]: Please wrap text in i18n 🙇🏻 
Import useT from services/i18n and then do `const t = useT()` and then `t("Story")`

File: web/src/beta/features/Editor/SidePanel/index.tsx
Comment (KaWaite) [inline web/src/beta/features/Editor/SidePanel/index.tsx:1]: For the `SidePanel` is there any chance that the order will change? Or in certain circumstances some components won't be used? 
Why I ask, is of course we can export each separately and use them like `SidePanel.Title`, `SidePanel.Wrapper` etc, but if we don't need the customization, keeping it a single component will make writing all side panels easier.

File: web/src/beta/features/Editor/Tabs/Storytelling/StorySidePanel/index.tsx (hunk 1 above)
Comment (isoppp) [reply web/src/beta/features/Editor/Tabs/Storytelling/StorySidePanel/index.tsx]: Thank you fixed :pray: https://github.com/reearth/reearth/pull/565/commits/7177f8a5056938ff1e0b0fa2add09904ae883efa

File: web/src/beta/features/Editor/SidePanel/index.tsx
Comment (isoppp) [reply web/src/beta/features/Editor/SidePanel/index.tsx:1]: Adjusted in https://github.com/reearth/reearth/pull/565/commits/24ca086afb64a11fcb40e131adcf208f7f15c140 .

For now, there is no plan to customize.
I was thinking I have to do [this kind of code](https://github.com/reearth/reearth/pull/565/commits/24ca086afb64a11fcb40e131adcf208f7f15c140#diff-50d3a40978836b8c5f1ccd08f7e347b0c6e22549bba2ff6e9bbe5285003ed562) in `useLeftPanel` hooks but I noticed I don't have to do in hooks, so single component is probably ok now.

**Review by KaWaite — APPROVED** [review]

File: web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  onStoryAdd: () => void;
+  selectedPageId?: string;
+  onSelectPage: (id: string) => void;
+  onPageAdd: () => void;
+};
+
+const SidePanel: FC<Props> = ({ onStoryAdd, onSelectStory, onPageAdd, onSelectPage }) => {
+  const t = useT();
+
+  const StoryContent = useMemo(() => {
```
Comment (pyshx) [inline web/src/beta/features/Editor/tabs/story/SidePanel/index.tsx:25]: Can you extract the StoryContent and PageContent components to separate functional component?

**Review by pyshx — APPROVED** [review]
Small Suggestion. Rest LGTM!

**Pull request merged**